README.md

```diff
@@ -1,124 +1,86 @@
-
-```bash
-git branch
+```sh
+yarn format
 ```
 
-You should see a single branch listed, called `gatsby`. If so, that's good! This is the development branch for the project and code should only ever be pushed to or pulled from there.
+### Git
 
-## Node & Yarn
+Once you have your editor set up, it is important that you're working on the correct branch. As github uses the `master` branch for deployment, we use the branch `gatsby` as the main development branch. Git should recognize this and automatically check out `gatsby` for new clones. You can check which branch you're on with `git branch`. The branch with the `*` before it is the currently selected branch. You can use `git checkout <branch-name>` to switch branch.
 
-**Node**
-The CyberSoc website requires Node to run and be developed properly, so make sure it is installed. You can check that everything has been installed correctly by typing:
+### Node
 
-```bash
-node -v
-```
+The website requires [node.js](https://nodejs.org/) to be installed to run properly. Install this using your preferred method. You can check that node is installed properly by running the command `node -v`. This command will print the version of node installed. At the time of writing the site, that was version `v12.16.1`, so we treat that as the minimum supported version.
 
-This will list the current version of Node running on your computer. At the time of writing that version was `v12.16.1`, so anything the same or higher is a good thing.
+### Yarn
 
-**Yarn**
-Next, we want to install Yarn. By default Node comes with a package manager called NPM (Node Package Manager) but due to personal preference when starting the project I'd strongly recommend continuing with Yarn (this is due to how the package managers keep track of and "lock" the dependencies; using multiple at once could cause problems).
+We use the [yarn](https://yarnpkg.com/) package manager for dependency management. Theoretically it is possible to use `npm`, but due to issues with dependency locking it is strongly discouraged.
 
-Yarn is used to install any and all third-party packages used for this project (and, it being a Node project, there are _quite_ a few). The list of dependencies for the website can be found in the `package.json` file in the root of the project directory.
+Install yarn using your preferred method. You can check that yarn is installed correctly by running `yarn -v`. This command will print the version of yarn installed.
 
-To automatically install all the required dependencies, get that terminal open and head back to the `cybersocyork.github.io` directory. Once inside, run:
+Once you have yarn installed, you can use it to download the (many) dependencies. Run the following from a terminal:
 
-```bash
+```sh
 yarn install
 ```
 
-This might take an amount of time to complete, so go and make a cup of coffee while you wait :)
```

Comment:
Removing all the personality from this README 😭

Reply:
:sob: it looks more professional though

Reply:
I understand. Probably something I carried across from personal repos lol